R/Sound_DMR.R

```diff
@@ -1464,15 +1461,16 @@ generate_zoomframe <- function(gene_cord_df, MFrame, Gene_col, target_info=TRUE,
 #' @examples
 #' # Basic usage for methyl_call_type
 #' # 1. With gene coordinate file
-#' # generate_methylframe(methyl_bed_list= <list_of_BedMethyl files>, gene_info = TRUE, gene_cordinate_file = <File with gene info>, Gene_col=<Gene Name/Gene ID>, target_info=TRUE, gene_list = <list of target genes>)
+#' # generate_methylframe(methyl_bed_list= <list_of_BedMethyl files>, gene_info = TRUE, gene_cordinate_file = <File with gene info>, 
+#' #                       Gene_col=<Gene Name/Gene ID>, target_info=TRUE, gene_list = <list of target genes>)
 #' # 2. Without gene cooridnate file
 #' # generate_methylframe(methyl_bed_list= <list_of_BedMethyl files>, gene_info = FALSE )
 #' @export
 
 generate_methylframe <-function(methyl_bed_list=All_methyl_beds, Sample_count = 0,
                                 Methyl_call_type="Dorado", filter_NAs=0, max_read_depth=100,
                                 gene_info = FALSE, gene_coordinate_file = NULL, Gene_column='',
-                                target_info=FALSE, gene_list = gene_coordinate_file[[Gene_column]],
+                                target_info=TRUE, gene_list = gene_coordinate_file[[Gene_column]],
```

Comment:
Why did we change the default here to true?

Reply:
TRUE makes it use the default gene_list parameter according and create a target_info column based on how the function is written. False doesn't throw any error but just skips making the target_info column which is not what we want by default.